### PR TITLE
[39] Next Step Page - Ministry Team

### PR DESCRIPTION
**Pull request by thedomcab**
image

### CODE REVIEW

File: frontend/src/images/ministry-teams.png
Comment (hmccaa) [inline frontend/src/images/ministry-teams.png:1]: this photo is misnamed - it's a UGR photo

**Review by hmccaa — APPROVED** [review]

**Review by xumaple — APPROVED** [review]

**Comment by thedomcab** [comment]
Resolve merge conflict.  need re-approval.

**Review by xumaple — APPROVED** [review]
Looks good. Is it possible to make the css circle work for a variable number of ministry teams? If so, let's create a work item to refactor that later

**Comment by thedomcab** [comment]
> Looks good. Is it possible to make the css circle work for a variable number of ministry teams? If so, let's create a work item to refactor that later

of course. this controls the placement in the circles:  
image

**Comment by thedomcab** [comment]
needs re-approval because i had to resolve conflicts.

**Pull request merged**